modules/core/map/farm_map.post_update.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Post update hooks for the farm_map module.
+ */
+
+use Drupal\farm_map\Entity\MapBehavior;
+use Drupal\farm_map\Entity\MapType;
+
+/**
+ * Generalize geofield map types and behavior.
```

Comment:
Is there a pattern / best practice of linking back to the relevant issue here or do we just rely on the git history to provide the justification for these?

Reply:
I've just relied on Git to provide the context. These `[module].post_update.php` files will be relatively linear in their histories (only additions, no refactors, no moving code between files), so `git blame` can get you to the history you need fairly easily. But it's a good point/question to raise, and maybe we should start including the relevant issue in comments for reference.